tests/yam/validation/pages/storage-page.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class StoragePage {
+    readonly page: Page;
+    readonly encryptionSlider: Locator;
+    readonly userPassword: Locator;
+    readonly userPasswordConfirm: Locator;
+    readonly acceptButton: Locator;
+    readonly encryptionEnabled: Locator;
+    readonly backButton: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.encryptionSlider = page.locator('#encryption-off');
+        this.userPassword = page.getByRole('textbox', { name: 'User password', exact: true });
```

Comment:
is not a user involved here perhaps internally is called like that but might be a bug.

Reply:
`User password` is what the Playwright generator recorded...

Reply:
yes, that is fine, the internal ids could be really weird, let's use the text that is visible to the user in the screen to create the name of the variable.